README.md

```diff
@@ -1,114 +1,147 @@
+<dl>
+<dt><a href="#get">get(pids, callback)</a></dt>
+<dd><p>Get pid informations.</p>
+</dd>
+</dl>
 
-> Maximum number of bytes in the working set of this process at any point in time. The working set is the set of memory pages touched recently by the threads in the process. If free memory in the computer is above a threshold, pages are left in the working set of a process even if they are not in use. When free memory falls below a threshold, pages are trimmed from working sets. If they are needed, they are then soft-faulted back into the working set before they leave main memory.
+### Typedefs
 
-The CPU usage is computed the same as it is on linux systems. We have the `kernelmodetime` and the `usermodetime` processor use. Every time `pidusage.stat` is called, we can calculate the processor usage according to the time spent between calls (uses `os.uptime()` internally).
+<dl>
+<dt><a href="#pidCallback">pidCallback</a> : <code>function</code></dt>
+<dd></dd>
+</dl>
 
-Note that before we used `wmic path Win32_PerfFormattedData_PerfProc_Process WHERE IDProcess=` (which is slow as hell) and `Win32_PerfRawData_PerfProc_Process` (which api breaks on Windows 10 and Windows server 2012). Not every Windows bugged but just some of those. However, the `wmic PROCESS` call is faster.
+<a name="get"></a>
 
-#### pidusage-tree
+##### get(pids, callback)
+Get pid informations.
 
-If you want to compute a pidusage tree take a look at [pidusage-tree](https://github.com/soyuka/pidusage-tree).
+**Kind**: global function  
+**Access**: public  
 
-#### pidusage-promise
+| Param | Type | Description |
+| --- | --- | --- |
+| pids | <code>Number</code> \| <code>Array.&lt;Number&gt;</code> \| <code>String</code> \| <code>Array.&lt;String&gt;</code> | A pid or a list of pids. |
+| callback | [<code>pidCallback</code>](#pidCallback) | Called when the statistics are ready. |
 
-Need promise? Use [pidusage-promise](https://github.com/soyuka/pidusage-promise)!
+<a name="pidCallback"></a>
 
-#### Legacy
+#### pidCallback : <code>function</code>
+**Kind**: global typedef  
 
-Prior 2.0.0, on linux procfiles where used. It has been removed due to performance issues when reading files. Indeed, `ps` is faster.
+| Param | Type | Description |
+| --- | --- | --- |
+| err | <code>Error</code> | A possible error. |
+| statistics | <code>Object</code> | The object containing the statistics. |
 
-Benchmark:
+## Related
+- [pidusage-tree][gh:pidusage-tree] -
+Promisified version of pidusage
+- [pidusage-tree][gh:pidusage-tree] -
+Compute a pidusage tree
 
-```
-Benching 246 process
-NANOBENCH version 2
-> node test/bench.js
+## Authors
+- **Antoine Bluchet** -  *Follow* me on *Github* ([:octocat:@soyuka][github:soyuka]) and on  *Twitter* ([🐦@s0yuka][twitter:s0yuka])
+- **Simone Primarosa** -  *Follow* me on *Github* ([:octocat:@simonepri][github:simonepri]) and on  *Twitter* ([🐦@simoneprimarosa][twitter:simoneprimarosa])
 
```

Comment:
I really don't agree to add things like this on the readme its point being to explain the code not to sell myself or any of the collaborators.

Reply:
No preference over this. I just used my default template 👍